amazonka/src/Network/AWS/Auth.hs

```diff
@@ -175,11 +180,12 @@ data Credentials
     | Discover
       -- ^ Attempt credentials discovery via the following steps:
       --
-      -- * Read the 'envAccessKey' and 'envSecretKey' from the environment if they are set.
+      -- * Read the 'envAccessKey', 'envSecretKey', and 'envRegion' from the environment if they are set.
```

Comment:
No chance that AWS_REGION be read from `~.aws/config` as part of the discovery as well? Or is that just a CLI thing?